examples/iOS/SimpleImageFilter/SimpleImageFilter/ViewController.swift

```diff
@@ -29,5 +28,31 @@ class ViewController: UIViewController {
         picture --> grainFilter --> renderView
         picture.processImage()
     }
+
+    private func applyKiraKira() {
+        let filter = {
+            let filter = Kirakira(rayCount: 3)
+            filter.equalMinHue = 0.54
+            filter.equalMaxHue = 0
+            filter.equalSaturation = 0.15
+            filter.equalBrightness = 2.8
+            filter.rayLength = 0.5
+            filter.sparkleExposure = 0.1
+            filter.blur = 0
+            filter.colorMode = .random
+            filter.saturation = 0.5
+            filter.centerSaturation = 0.3
+            filter.minHue = 0
+            filter.maxHue = 1
+            filter.noiseInfluence = 1
+            filter.increasingRate = 0.03
+            filter.sparkleAmount = 0.6
```

Comment:
idea:

Create a Codable struct so that we can decode the Kirakira JSON and pass the parameters object into Kirakira's initializer to set up the Kirakira effect.

```swift
class Kirakira {
    public struct Parameters: Codable {
        // Define properties here...
    }
}
```

Reply:
Hi @yyjim, I'll merge this PR first, and then create another one to support this implementation.
I've cleaned up the code that you've left comments on, but please feel free to leave comments in the next PR if you have any other concerns in this one; thanks!